templates/strings/flat-swift4.stencil

```diff
@@ -70,8 +70,7 @@ import Foundation
 
 extension {{enumName}} {
   private static func tr(_ table: String, _ key: String, _ args: CVarArg...) -> String {
-    // swiftlint:disable:next nslocalizedstring_key
-    let format = {{param.localizeFunction|default:"NSLocalizedString"}}(key, tableName: table, bundle: BundleToken.bundle, comment: "")
+    let format = BundleToken.bundle.{{param.localizeFunction|default:"localizedString"}}(forKey: key, value: nil, table: table)
```

Comment:
```suggestion
    let format = {{param.localizeFunction|default:"BundleToken.bundle.localizedString"}}(forKey: key, value: nil, table: table)
```

When you want a custom localisation function it's usually because you want to look for localisation in the main bundle first before the fmk bundle, or in a separate bundle, etc… so making the template calling the function on the instance of `BundleToken.bundle` kinda defeats the idea 😉

Reply:
Another advantage of going that route, is that people wanting to just replace the default behaviour to instead just look at localisation in the main bundle, can just pass `Bundle.main.localizedString` for the `localizeFunction` without having to implement a dedicated function. Or `ResourceBundle.bundle.localizeFunction` if they want to make the behaviour look at localization in a resources bundle, without having to declare a custom method implementation for that.